src/OpenLoco/src/Vehicles/VehicleBody.cpp

```diff
@@ -887,22 +887,22 @@ namespace OpenLoco::Vehicles
             case SimpleAnimationType::steam_puff1:
             case SimpleAnimationType::steam_puff2:
             case SimpleAnimationType::steam_puff3:
-                steamPuffsAnimationUpdate(1, var_05);
+                steamPuffsAnimationUpdate(1, emitterHorizontalPos);
```

Comment:
for all of these functions they have named the 2nd parameter var_05 they should all be updated to emitterHorizontalPos.